Displaying experimental timer changes

PackSciences opened the pull request:
I modified the display of the experimental timer .
It displays in green (instead of the disgusting red WARNING) the total ticks , then displays it into a "normal" time (hours, minutes, seconds, milliseconds) .

Idea for an update :
I had an idea about avoiding the experimental getting bypassed .
To do it , I thought SPT could display a number that would be half random half based on the total time to prevent cheated runs .

YaLTeR commented:
Green color display: yes, that's probably a good idea.
Total time: no, there's no good way of getting the current tickrate without finding a whole load of patterns for a whole load of things.
Idea for an update: what, I don't even understand.

PackSciences commented:
Where are the errors ?
Idea for an update : It would be a "verifying code" that would be crypted and confirm that the run is accurate in case someone submits a run with Photoshop or something.

PackSciences commented:
Edited pull request
Removed : Time conversion
Added : Changed Warning into Green text displays + Some functions text displays for a better use of the timer functions.

PackSciences commented:
Edit

lipsanen commented:
it has been done
